BMM.Core/Translation/en/main.json

```diff
@@ -281,7 +281,9 @@
     "Podcasts": "Podcasts",
     "Playlists": "Playlists",
     "NoResults": "No results",
-    "NoResultsDescription": "There where no matches with your search for \"{0}\""
+    "NoResultsDescription": "There where no matches with your search for \"{0}\"",
+    "SearchFailedTitle": "Search failed",
+    "SearchFailedMessage": "Sorry, we could not search for the given phrase due to connection issues. Press the button below to try again."
```

Comment:
Sorry, we could not search for "{0}" because of connection issue. Press the button below to try again.